controllers/actions.github.com/autoscalingrunnerset_controller.go

```diff
@@ -154,7 +153,7 @@ func (r *AutoscalingRunnerSetReconciler) Reconcile(ctx context.Context, req ctrl
 	}
 
 	// Make sure the runner group of the scale set is up to date
-	currentRunnerGroupName, ok := autoscalingRunnerSet.Annotations[runnerScaleSetRunnerGroupNameKey]
+	currentRunnerGroupName, ok := autoscalingRunnerSet.Annotations[LabelKeyGithubRunnerGroupName]
```

Comment:
change all `Github` -> `GitHub`? 😄

Reply:
I'll rename it :relaxed: Thanks for the suggestion!